--- origsrc/luasocket-2.0.2/src/makefile	2007-10-14 23:21:05.000000000 -0500
+++ src/luasocket-2.0.2/src/makefile	2008-02-13 12:42:27.718750000 -0600
@@ -50,13 +50,13 @@
 all: $(SOCKET_SO) $(MIME_SO) 
 
 $(SOCKET_SO): $(SOCKET_OBJS)
-	$(LD) $(LDFLAGS) -o $@ $(SOCKET_OBJS)
+	$(LD) $(LDFLAGS) -o $@ $(SOCKET_OBJS) $(LIBS)
 
 $(MIME_SO): $(MIME_OBJS)
-	$(LD) $(LDFLAGS) -o $@ $(MIME_OBJS)
+	$(LD) $(LDFLAGS) -o $@ $(MIME_OBJS) $(LIBS)
 
 $(UNIX_SO): $(UNIX_OBJS)
-	$(LD) $(LDFLAGS) -o $@ $(UNIX_OBJS)
+	$(LD) $(LDFLAGS) -o $@ $(UNIX_OBJS) $(LIBS)
 
 #------
 # List of dependencies
